projects/playground/src/app/app.module.ts

```diff
@@ -17,6 +17,7 @@ import {
 const authConfig: AuthConfig = {
   clientId: 'wLSIP47wM39wKdDmOj6Zb5eSEw3JVhVp',
   domain: 'brucke.auth0.com',
+  skipRedirectCallback: window.location.pathname === '/other-callback',
```

Comment:
Do we need to have a route set up to handle this? How does this work in the playground from a user's perspective?

Reply:
If you want to use this, yes you do.
I added it to the playground as a matter of showing how to configure it, but we do not realy integrate our playground with another OAuth provider tho.

Reply:
In that case it might be better to instead show a usage example in the docs against the new property - I can imagine people will look there first before they dive into the playground code.

Do you think this also merits a section in the readme about usage with other providers?

Reply:
Yes I think that is a good idea and I updated the PR. 

![image](https://user-images.githubusercontent.com/2146903/99513881-65cc1a80-298b-11eb-8718-e0c0d3143354.png)


![image](https://user-images.githubusercontent.com/2146903/99513825-56e56800-298b-11eb-8c12-228eedb9b967.png)